standards/prettier/.prettierrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "trailingComma": "all",
+  "printWidth": 80,
+  "tabWidth": 4,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": false,
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "arrowParens": "always",
+  "requirePragma": false,
+  "insertPragma": false,
```

Comment:
What do these two rules do? Do they only format the files that have the pragma on top of the file or have I misunderstood that?

Reply:
require pragma requires that you have prettier comment on top of a file to format it.
``` 
/**
 * @prettier
 */
``` 

Insert pragma inserts the comment after formatting. We don't need any of these.